frontend/css/theme/bootstrap/mixins/buttons.less

```diff
@@ -59,6 +59,15 @@
   }
 }
 
+.searchFieldCopyButton{
```

Comment:
I see two small issues here. One is that this is the wrong file to add this CSS: everything in the `frontend/css/theme/bootstrap/` folder is stock Bootstrap files.
All our custom css/less is in the `frontend/css/` folder instead.


Here is what I suggest to match what we currently do for other modals:
1. change the modal's topmost component's `id` to match what we do for other modal components -> `"MBIDMappingModal"`:
https://github.com/metabrainz/listenbrainz-server/blob/7681c8d8e63883b42eea202d4d495ae8221da741/frontend/js/src/personal-recommendations/PersonalRecommendationsModal.tsx#L164
2. create a new `mbid-mapping-modal.less` file
3. Move your less code to the new `mbid-mapping-modal.less` file , under `#MBIDMappingModal` block like we do for the other modals:
https://github.com/metabrainz/listenbrainz-server/blob/7681c8d8e63883b42eea202d4d495ae8221da741/frontend/css/personal-recommendation-modal.less#L3-L4
4. rename `searchFieldCopyButton` to `search-field-copy-button` to match the rest of our css classes names.